### PR TITLE
First draft FAQs

### PR DESCRIPTION
**Pull request by DanP20**
pushing for team review

### CODE REVIEW

File: src/guides/FAQs.md [hunk 1]
```diff
@@ -0,0 +1,163 @@
+# NodeScript FAQs
```
Comment (andrew-waters) [inline src/guides/FAQs.md:1]: Can we rename this file to `faqs.md` - all filenames should be lower case

File: src/.vuepress/config.js [hunk 2]
```diff
@@ -72,6 +72,7 @@ module.exports = {
             // 'consuming-gcp-pub-sub',
             // 'building-a-slack-application',
             'weather-bot',
+            'FAQs',
```
Comment (andrew-waters) [inline src/.vuepress/config.js]: this will need updating to `faqs`

**Review by andrew-waters — CHANGES_REQUESTED** [review]

File: src/guides/FAQs.md (hunk 1 above)
Comment (andrew-waters) [reply src/guides/FAQs.md:1]: Did this get renamed or is it still mixed case?

Comment (DanP20) [reply src/guides/FAQs.md:1]: I would keep the Frontend page title as FAQ - but all the internal file names and links are lower

Comment (andrew-waters) [reply src/guides/FAQs.md:1]: Yep, it's not this line I'm referencing, it's the actual filename

**Review by andrew-waters — APPROVED** [review]

**Pull request merged**